client/influxdb.go

```diff
@@ -252,11 +274,11 @@ func (a Results) Error() error {
 // Precision can be specified if the timestamp is in epoch format (integer).
 // Valid values for Precision are n, u, ms, s, m, and h
 type Point struct {
-	Name      string
-	Tags      map[string]string
-	Timestamp time.Time
-	Fields    map[string]interface{}
-	Precision string
+	Name      string                 `json:"name,omitempty"`
```

Comment:
is it really that important to remove these tags? How do you want to notate that the tags on time don't work?